Jenkinsfile

```diff
@@ -3,16 +3,12 @@
 node {
     checkout scm
 
-    nodeEnv = docker.image("node:10")
+    nodeEnv = docker.image("node:10-stretch")
     workspace = pwd()
-    pkgVersion = sh (
-      script: 'cat package.json | jq -r .version',
-      returnStdout: true
-    ).trim()
```

Comment:
This got moved to the end of the script, since the version number changes during the deployment process.